### DISCUSSION TITLE
Selling with unintend price

### DISCUSSION THREAD

**Discussion by longfin** [planetarium/NineChronicles#1873] [Ideas]
Note: sake for comms, the comments of this discussion may be written in Korean. (you can use translate feature).

![image](https://user-images.githubusercontent.com/128436/187432503-b7aa83b8-fd7a-4dd1-a515-ef2e5d1660c3.png)
(Captured on v100282)

- Current sell UI isn't intuitive enough, to recognize price and quantity. 
- Moreover, since there is no guard until this is confirmed, users frequently have problems with selling and registering items with unintended NCG.
- Thus we need to perform these actions I guess...
   - Improving UX to recognize price easily. (sadly, I couldn't prepare suggestions due to limited UX knowledge 😢)
   - Adding confirmation between that UI and sedning action.

Any ideas? @planetarium/nine-chronicles

**Comment by BBeom06** [comment]
보내주신 내용에 대해서는, 내부에서 인지하고 있는 부분이여서
단일(장비)상품과 묶음상품에 대한 인지부분을 차별화를 해서 수정할 계획을 가지고 있습니다.
의견 감사드립니다.

**Reply by longfin** [reply]
@BBeom06 

- 혹시 관련해서 예정된 작업 완료 일시가 있을까요?
- 말씀하신 단일/묶음 상품을 잘 보여주는 것 외에도, 액션을 실제로 보내기 전에 확인 창이 있었으면 합니다. 혹시 이 부분에 대해서도 고려되고 있을까요? cc @Namyujeong

**Reply by longfin** [reply]
> 액션을 실제로 보내기 전에 확인 창이 있었으면 합니다. 

여기에 대해서 여러 반대 의견 등이 있었던 걸로 압니다. 혹시 어떤 우려나 문제점들이 있었는지도 적어주실 수 있을까요? @planetarium/nine-chronicles

**Comment by BBeom06** [comment]
다음 주쯤, 예정된 작업은 완료할 예정입니다.

판매 등록에 관련한 확인 창의 경우는,
[유저의 실수]와 [편의]라는 것에 어느쪽에 더 무게를 두어야 하는가를 두고 등록 절차를 간소화하는 방향성과 다른 접근 방식이였습니다.
가격 입력 단계에서 실수가 없도록 하는 방안으로 발전 시키고자 하는 생각은 가지고 있었으나,

유저의 실수를 보완하고자 하는 부분이 더 높은 방향성이라면 등록에 대한 절차를 추가하고자 합니다.
확인 팝업에 결과값(아이템,판매가격,개당가격,수수료) "아이템을 등록 하시겠습니까?" 라는 물음 형태로 단계를 추가해서 수정 및 보완을 해보고자 합니다.

**Comment by BBeom06** [comment]
해당 내용은 진행됐습니다.
내부적으로, 두번째 스탭으로 확인이 필요하도록 하는것보다는 첫번째 스텝에서 해당 항목에 대해 실수를 주지 않는 방향에 맞춰 개선이 진행됐습니다.

1. 금액 입력란에는 NCG 아이콘을 추가해서, 수량과 금액에 대한 차이를 주었습니다.
2. 수량입력과 판매등록의 버튼의 영역을 늘려 입력 오류로 인해 등록하는 실수를 줄이고자 했습니다.
